swagger.go

```diff
@@ -22,20 +23,33 @@ type Swagger interface {
 func Register(name string, swagger Swagger) {
 	swaggerMu.Lock()
 	defer swaggerMu.Unlock()
+
 	if swagger == nil {
 		panic("swagger is nil")
 	}
 
-	if swag != nil {
+	if swags == nil {
+		swags = make(map[string]Swagger)
+	}
+
+	if _, ok := swags[name]; ok {
 		panic("Register called twice for swag: " + name)
 	}
-	swag = swagger
+	swags[name] = swagger
 }
 
 // ReadDoc reads swagger document.
 func ReadDoc() (string, error) {
-	if swag != nil {
+	return ReadDocName(Name)
+}
+
+// ReadDocName reads swagger document with specific name.
+func ReadDocName(name string) (string, error) {
```

Comment:
Changing the function signature is not an acceptable solution. This is a breaking change and breaks all other projects.
`func ReadDoc() (string, error) {`
to 
`func ReadDocName(name string) (string, error) {`